test/e2e-tests.sh

```diff
@@ -153,7 +153,7 @@ toggle_feature secure-pod-defaults Disabled
 
 # Run HA tests separately as they're stopping core Knative Serving pods.
 # Define short -spoofinterval to ensure frequent probing while stopping pods.
-go_test_e2e -timeout=25m -failfast -parallel=1 ./test/ha \
+go_test_e2e -timeout=40m -failfast -parallel=1 ./test/ha \
```

Comment:
Could we make the timeout smaller or this is the lower bound for test to pass.
I see that it:
```
PASS test/ha.TestActivatorNotInRequestPath (326.48s)
```

Reply:
I have tried 25m as it was before, but the test times out. I'll incrementally increase to find out the lower bound for it to pass

Reply:
It looks like the tests take somewhere between 25 and 30min, so changed it to 30min